runtime/testy/runtime.py

```diff
@@ -91,17 +91,18 @@ def nonTestModePrint(*args):
           controlState = "auto"
           continue
         elif newBadThing.event == BAD_EVENTS.ENTER_IDLE and controlState != "idle":
+          controlState = "idle"
```

Comment:
We will address this in #153 to properly kill studentCode Process when swapping modes other than IDLE. (IDLE here should kill the student code.)